tests/common/devices.py

```diff
@@ -227,22 +232,26 @@ def get_pmon_daemon_list(self):
         logging.info("Pmon daemon list for this platform is %s" % str(daemon_list))
         return daemon_list
 
-    def get_up_time(self):
-        up_time_text = self.command("uptime -s")["stdout"]
-        return datetime.strptime(up_time_text, "%Y-%m-%d %H:%M:%S")
+class FanoutHost(Localhost):
```

Comment:
It looks like there is a better way to implement the Cisco like cli devices. Please refer to below example which use the ansible builtin network_cli network connection plugin. It looks like a more 'ansible' way than hacking using 'ansible-playbook' cli.
https://github.com/Azure/sonic-mgmt/pull/1475/files#diff-388cd8aa68de5bbbd57b2aafe842dccf

Reply:
For now it is just given possibility to execute playbook on Fanout host.
Currently it is enough.

When there will be a need to do more, then it should be extended with new functionality.
As I know in different customers on Fanout device can be different OS and different CLI if such available. So this should be taken into consideration while development.

Agree?

Reply:
By the way in such implementation ansible playbook can be executed for different vendors with its own commands.

Reply:
In that case, the class should be more generic, not just for fanout. It's just for running a playbook on localhost.

Reply:
Moved this class to the specific conftest of pfc_asym test case.
To avoid creation of some generic class in the current devices.py file.